support/cas-server-support-passwordless-ldap/src/test/java/org/apereo/cas/impl/account/LdapPasswordlessUserAccountStoreTests.java

```diff
@@ -35,7 +35,8 @@
     "cas.authn.passwordless.accounts.ldap.bind-dn=cn=Directory Manager",
     "cas.authn.passwordless.accounts.ldap.bind-credential=password",
     "cas.authn.passwordless.accounts.ldap.email-attribute=mail",
-    "cas.authn.passwordless.accounts.ldap.phone-attribute=telephoneNumber"
+    "cas.authn.passwordless.accounts.ldap.phone-attribute=telephoneNumber",
+    "cas.authn.passwordless.accounts.ldap.request-password=false"
```

Comment:
This is invalid. There is no such setting.

Reply:
sorry I tried I'm not a developper but I tried this with cas-overlay and it was working. What am I missing ? 
Modification of [PasswordlessAuthenticationLdapAccountsProperties.java](https://github.com/apereo/cas/pull/5600/commits/b81c899661a211cfab2c37b8a31963121250dcbb#diff-f2ff177bc35aed8a57d5658bd2c4d96e86d42a04e6dd36b551612be9d6df2670)  does not add the settings ?

Reply:
It does. Your setting does not match what you added. Is this setting actually called `request-password`?

Reply:
ok sorry about that I completely misunderstood the way the test was working. 
Then I should load a ldap extension with an attribute requestPassword for example
Set it to true in the ldif
And map it in my config here...

Reply:
Yes correct, you got it.